src/components/UserForm/UserForm.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { FC, useState, ChangeEvent } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import './UserForm.css';
+import { CompanyInterface, UserInterface, ValidationErrors } from '../../Interfaces/ObjectInterfaces';
+import { formValidation } from '../../services/formValidation';
+import { guIdGenerator } from '../../services/guidGenerator';
+
+export const UserForm: FC = () => {
+    let navigate = useNavigate();
+    let fetchedUserList = window.localStorage.getItem("storedUserList");
+    let fetchedCompanyList = window.localStorage.getItem("storedCompanyList");
+    let fetchedPositionList = window.localStorage.getItem("storedPositionList");
```

Comment:
If you mean 'position list', yes it can be hard coded in an array. In the beginning I thought maybe there will be some option for the user to edit positions in the app.